inc/osvr/Util/ProcessUtils.h

```diff
@@ -0,0 +1,91 @@
+/** @file
+@brief Header
+
+@date 2016
+
+@author
+Sensics, Inc.
+<http://sensics.com/osvr>
```

Comment:
I did use the generator but copied and pasted. VS borked up the formatting. I'll fix.